meshroom/ui/qml/GraphEditor/Edge.qml

```diff
@@ -148,8 +148,21 @@ Item {
         Component.onCompleted: {
             /* The curve scale must be set only once the component has been fully created, so
              * that all the events following the update of the curve scale can be taken into
-             * account */
+             * account. */
             curveScale = cubic.ctrlPtDist / root.width  // Normalize by width
         }
+
+        Connections {
```

Comment:
If I understand the problem well, Qt6 broke the declarative binding we used to have for the `curveScale`, because the order in which some events are triggered has changed.
If so, I would recommend keeping the logic in `Component.onCompleted`, but create that binding imperatively using `Qt.binding()`. Instead of having a first initialization there, and the updates handled in a Connections component.

Something like:
```
Component.onCompleted: {
    /* ... */
    curveScale = Qt.binding(() => cubic.ctrlPtDist / root.width)  // Normalize by width
}
```

Reply:
Thank you for this feedback! All done :)